fuzz/README.md

```diff
@@ -85,9 +108,5 @@ The final line is a hex-encoded version of the input that caused the crash. You
 can test this directly by editing the `duplicate_crash` test to copy/paste the
 hex output into the call to `extend_vec_from_hex`. Then run the test with
 
-    RUSTFLAGS=--cfg=fuzzing cargo test
-
-It is important to add the `cfg=fuzzing` flag, which tells rustc to compile the
-library as though it were running a fuzztest. In particular, this will disable
-or weaken all the cryptography.
+    cargo test
```

Comment:
Is this removed because of spurious bug reports?

Reply:
It's removed because now you have to take action to set your RUSTFLAGS. Before you had to know that honggfuzz was setting this flag, that our code would behave differently because of it, and that you need to set it manually for `cargo test` to reproduce failures.

Reply:
Oh, I understand now. OK.